README.adoc

```diff
@@ -211,63 +270,66 @@ You will see a login form since form authentication is implemented and configure
 
 Notice that when you sign in as Bob or Carl, the browser redirects to the `admin` page and you can view their names and roles.
 When you sign in as Alice, you can only view Alice's name.
-When you sign in as Dave, you'll be blocked and will see a `Error 403: Authorization failed` message because Dave doesn't have a role supported by the application.
+When you sign in as Dave, you'll be blocked and will see a `Error 403: Authorization failed` message 
+because Dave doesn't have a role supported by the application.
 
-Once you're done checking out the application, log out and stop the Open Liberty server:
-[role='command']
-```
-mvn liberty:stop-server
-```
 
 // =================================================================================================
 // Testing the application
 // =================================================================================================
 
 == Testing the application
 
-Write `SecurityTest` class to test the authentication and authorization of the application.
+Write [hotspot file=0]`SecurityIT` class to test the authentication and authorization of the application.
```

Comment:
Update to:
"Write the [hotspot file=0]`SecurityIT` class to test the authentication and authorization of the application."